styles.css

```diff
@@ -1 +1,601 @@
+    font-family: 'Merriweather', serif;
+    color: var(--primary-color-purple);
+    font-size: 24px;
+    padding: 200px 30px 30px 30px;
+
+}
+
+.question {
+    font-family: 'Roboto', sans-serif;
+    color: var(--primary-color-black);
+    font-size: 24px;
+    font-weight: 500;
+    padding: 50px 30px 30px 30px;
+}
+
+ul > li > a {
+    text-decoration: none;
+    color: var(--primary-color-bleu);
+    font-weight: 500;
+}
+
+ul > li {
+    list-style: none;
+}
+
+/*------------- Begin "hallo" big fonts ------------------------*/
+.hero2-section div {
+    width: 650px;
+    height: 70px;
+    background-color: transparent;
+    border-radius: 10px;
+}
+
+.hero2-section div::after {
+    position: relative;
+    content: "hello";
+    font-family: 'Roboto', sans-serif;
+    font-weight: 900;
+    color: var(--primary-color-purple-light);
+    font-size: 340px;
+    display: block;
+    width: 0;
+    height: 0;
+    padding-left: 30px;
+    z-index: -1;
+}
+
+/*------------------End big fonts -------------------------*/
+
+/*------------- Begin "work" big fonts ------------------------*/
+.work {
+    width: 650px;
+    height: 70px;
+    background-color: transparent;
+    border-radius: 10px;
+    margin-right: 300px;
+    margin-top: 0;
+}
+
+.work::after {
+    position: relative;
+    content: "work";
+    font-family: 'Roboto', sans-serif;
+    font-weight: 700;
+    color: white;
+    font-size: 500px;
+    display: block;
+    width: 0;
+    height: 0;
+    padding-right: 30px;
+    margin-top: 70px;
+    z-index: 1;
+}
+
+/*------------------End big fonts -------------------------*/
+
+/*-------------Begin session WORK CSS----------------------*/
+.background {
+    background-color: #f3f6f9;
+    width: 100vw;
+}
+
+.top-work {
+    z-index: 2;
+}
+
+.shadow {
+    box-shadow: 0 24px 24px -5px #d3d1d1;
+}
+
+/*--------------------------- flexcontainer ----------------------------*/
+.flex-container {
+    flex-wrap: wrap-reverse;
+    margin-bottom: 60px;
+}
+
+.flex-container > div {
+    font-family: 'Merriweather', serif;
+    color: var(--primary-color-purple);
+    font-size: 24px;
+    width: 500px;
+    height: 300px;
+    margin-top: 230px;
+    margin-left: 20px;
+    margin-right: 20px;
+    z-index: 3;
+}
+
+.flex-container > div > h3 {
+    font-family: 'Merriweather', serif;
+    color: var(--primary-color-black);
+    font-size: 24px;
+    margin-top: -70px;
+}
+
+.flex-container > img {
+    width: 40%;
+    margin-bottom: 30px;
+    margin-top: 350px;
+}
+
+.flex-container > div > img {
+    margin-top: 10px;
+    margin-bottom: 70px;
+}
+
+.flex-container > div > p {
+    font-family: 'Roboto', sans-serif;
+    color: var(--primary-color-black);
+    font-size: 18px;
+    margin-bottom: 40px;
+}
+
+.flex-container > div > a {
+    font-family: 'Roboto', sans-serif;
+    color: var(--primary-color-black);
+    font-size: 24px;
+}
+
+/*-------------------- flex-container 2 --------------------------------*/
```

Comment:
De flex container die je nu hebt gemaakt is wat gelimiteerd, op het moment dat ik weer de mobiele versie erbij pak, verandert de hele layout van de website. De images groeien en krimpen niet goed mee. Denk hierbij aan het gebruik van flex grow & shrink. Dit geldt ook voor de icoontjes in het services blok! :)